rotkehlchen/externalapis/etherscan.py

```diff
@@ -285,11 +289,56 @@ def get_transactions(
             for entry in result:
                 gevent.sleep(0)
                 try:
-                    tx = deserialize_ethereum_transaction(  # type: ignore
-                        data=entry,
-                        internal=is_internal,
-                        ethereum=None,
-                    )
+                    # Handle genesis block transactions
+                    if is_genesis_tx(entry) is False:
```

Comment:
There is no need to create a function that is reading one dict field and is used in only one place.

Also what are you doing here exactly? This is not how they look like. The problem that prompted this entire issue is that the genesis transactions have a hash entry that is invalid. `GENESIS_XXXXXXX`

https://etherscan.io/txs?block=0

I mean the block would be zero, but the problem is the transaction hash

Reply:
Fixed (if I got your message correctly)